src/pages/studyView/charts/scatterPlot/StudyViewDensityScatterPlot.tsx

```diff
@@ -46,45 +48,14 @@ export interface IStudyViewDensityScatterPlotProps {
 const NUM_AXIS_TICKS = 8;
 const DOMAIN_PADDING = 15;
 
-class _VictorySelectionContainerWithLegend extends VictorySelectionContainer {
-    // we have to do this because otherwise adding the legend element messes up the
-    //  VictoryChart layout system
-
-    render() {
-        const {
-            activateSelectedData,
-            onSelection,
-            containerRef,
-            gradient,
-            legend,
-            children,
-            ...rest
-        } = this.props as any;
-        return (
-            <VictorySelectionContainer
-                activateSelectedData={false}
-                onSelection={onSelection}
-                containerRef={containerRef}
-                children={children
-                    .concat(legend)
-                    .concat(<defs>{gradient}</defs>)}
-                {...rest}
-            />
-        );
-    }
-}
-
-// need to do this for typescript reasons, because Victory isn't well typed
-const VictorySelectionContainerWithLegend = _VictorySelectionContainerWithLegend as any;
```

Comment:
moved to `packages/cbioportal-frontend-commons/src/components/victory/VictorySelectionContainerWithLegend.tsx`